mon-pix/app/pods/components/module/recap/styles.scss

```
@@ -0,0 +1,50 @@
+.module-recap {
+  max-width: var(--modulix-max-content-width);
+  margin: 0 auto;
+  padding: var(--pix-spacing-4x) var(--pix-spacing-4x) var(--pix-spacing-12x) var(--pix-spacing-4x);
+  background-color: var(--pix-primary-100);
+  border-radius: 16px;
+
+  &__header {
+    margin-bottom: var(--pix-spacing-6x);
+    text-align: right;
+  }
+
+  &__title, &__objectives {
+    margin-bottom: var(--pix-spacing-12x);
+  }
+
+  &__title {
+    @extend %pix-title-l;
+
+    color: var(--pix-neutral-900);
```

C'est déjà la couleur par défaut de `.modulix`

```suggestion
    color: var(--pix-neutral-900);
```